app/views/AryDashboard/WhatAssessed/GeographicalAreaAssessments/index.tsx

```diff
@@ -0,0 +1,316 @@
+        value: tooltipData.value,
+    }), []);
+
+    return (
+        <div className={_cs(className)}>
+            <SelectInput
+                placeholder="Select Region"
+                name="region"
+                value={regionValue}
+                onChange={setRegionValue}
+                keySelector={adminLevelKeySelector}
+                labelSelector={adminLevelLabelSelector}
+                options={data?.regions}
+                disabled={boundsPending}
+            />
+            <SegmentInput
+                className={styles.adminLevels}
+                name="adminLevels"
+                value={activeAdminLevel}
+                onChange={setActiveAdminLevel}
+                keySelector={adminLevelKeySelector}
+                labelSelector={adminLevelLabelSelector}
+                options={adminLevels ?? undefined}
+                disabled={navigationDisabled}
+            />
+            <Map
+                mapStyle={mapboxStyle}
+                mapOptions={mapOptions}
+                scaleControlShown={false}
+                navControlShown={false}
+            >
+                <MapContainer
+                    className={_cs(className, styles.map)}
+                />
+                {bounds && (
+                    <MapBounds
+                        bounds={bounds}
+                        padding={10}
+                        duration={100}
+                    />
+                )}
+                <MapSource
+                    sourceKey="regions"
+                    sourceOptions={sourceOptions}
+                    geoJson={adminLevelGeojson?.geojsonFile?.url ?? ''}
+                >
+                    <MapLayer
+                        layerKey="fill"
+                        layerOptions={fillLayerOptions}
+                        onMouseEnter={handleMouseEnter}
+                        onMouseLeave={handleMouseLeave}
+                    />
+                    <MapLayer
+                        layerKey="line"
+                        layerOptions={lineLayerOptions}
+                    />
+                    <MapState
+                        attributes={assessmentCountAttribute}
+                        attributeKey="assessmentCount"
+                    />
+                    {hoverLngLat && (
+                        <MapTooltip
+                            coordinates={hoverLngLat}
+                            tooltipOptions={tooltipOptions}
+                            trackPointer
+                            hidden={false}
+                        >
+                            <List
+                                data={hoverFeatureProperties}
+                                renderer={TextOutput}
+                                keySelector={keySelector}
+                                rendererParams={rendererParams}
+                            />
+                        </MapTooltip>
+                    )}
+                </MapSource>
+            </Map>
```

Comment:
hide map if source is not present instead of setting the url emtpy string.
Also make sure the UI doesn't jump